core/src/ledger/eth_bridge/storage/bridge_pool.rs

```diff
@@ -19,6 +19,7 @@ pub const BRIDGE_POOL_ADDRESS: Address =
     Address::Internal(InternalAddress::EthBridgePool);
 /// Sub-segment for getting the latest signed
 const SIGNED_ROOT_SEG: &str = "signed_root";
+const NONCE: &str = "bridge_pool_nonce";
```

Comment:
this should be refactored later when we merge 0.13. it includes the `StorageKeys` macro iirc